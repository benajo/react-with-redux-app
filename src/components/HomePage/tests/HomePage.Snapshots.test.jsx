import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import HomePage from '../HomePage';

it('render HomePage component', () => {
  const tree = renderer.create(
    <MemoryRouter>
      <HomePage />
    </MemoryRouter>,
  );
  expect(tree).toMatchSnapshot();
});
