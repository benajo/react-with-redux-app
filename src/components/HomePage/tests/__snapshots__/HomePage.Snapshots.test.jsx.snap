// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render HomePage component 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Pluralsight Administration
  </h1>
  <p>
    React, Redux and React Router for ultra-responsive web apps.
  </p>
  <a
    className="btn btn-primary btn-lg"
    href="/about"
    onClick={[Function]}
  >
    Learn more
  </a>
</div>
`;
